### ISSUE TITLE
mod_wsgi crashed with Segmentation fault (core dumped) in Alpine

### ISSUE THREAD

**Issue by svenikea** [GrahamDumpleton/mod_wsgi#656]
So I'm building an Apache container that runs with mod_wsgi for my Django application, the building process was fine until I run Apache on my container with ```httpd -D FOREGROUND``` and it gives me an error ```Segmentation fault (core dumped)```.
I install the package with ```apk add apache2-mod-wsgi``` and I put this line in apache config
```LoadModule wsgi_module modules/mod_wsgi-py27.so```
Any help with be great, thank you.

**Comment by GrahamDumpleton** [comment]
For now you will need to use ``mod_wsgi`` from Git repository on Alpine Linux. There has been a long standing bug which causes it to crash on Alpine Linux due to Alpine C libraries not quite being the same as glibc. It was fixed a while back but there hasn't been a recent release of ``mod_wsgi``.

Also suggest you not configure Apache yourself and instead use ``mod_wsgi-express``.

Thus:

```
pip install https://github.com/GrahamDumpleton/mod_wsgi/archive/develop.zip
```

and then use:

```
mod_wsgi-express start-server
```

See:

* http://blog.dscpl.com.au/2015/04/using-modwsgi-express-with-django.html
* http://blog.dscpl.com.au/2015/04/integrating-modwsgi-express-as-django.html

for how to use ``mod_wsgi-express`` with Django.

Also suggest watching:

* https://www.youtube.com/watch?v=CPz0s1CQsTE&t=6s

as to why it is better to use ``mod_wsgi-express``.

**Comment by GrahamDumpleton** [comment]
Closing as this was released in 4.8.0.
